lib/money/money.rb

```diff
@@ -1,77 +1,107 @@
-require 'bigdecimal'
-require 'bigdecimal/util'
-
 class Money
   include Comparable
 
-  attr_reader :value, :cents
+  attr_reader :value, :cents, :currency
 
   class << self
-    def new(value = 0, _currency = nil)
+    attr_accessor :parser, :default_currency
+
+    def new(value = 0, currency = nil)
```

Comment:
Why default to `nil` here, can you make the default `default_currency` in the method definition instead of the body? Or do we want to promote being able to pass a nil value in to indicate to use the default currency?